google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITQueryAggregationsTest.java

```diff
@@ -81,6 +81,25 @@ public void canRunCountUsingAggregationMethod() throws Exception {
     assertThat(snapshot.getCount()).isEqualTo(2);
   }
 
+  @Test
+  public void allowsAliasesLongerThan1500Bytes() throws Exception {
+    // The longest field name allowed is 1500. The alias chosen by the client is <op>_<fieldName>.
+    // If the field name is
+    // 1500 bytes, the alias will be longer than 1500, which is the limit for aliases. This is to
+    // make sure the client
+    // can handle this corner case correctly.
+    String longField = "";
+    for (int i = 0; i < 150; i++) {
+      longField += "0123456789";
+    }
```

Comment:
Can we use StringBuilder just like this example?
```
StringBuilder builder = new StringBuilder(N);
for (int i = 0; i < N; i++) {
    builder.append("a");
}
String newString = builder.toString();
```

Reply:
Done